pkg/apis/serving/v1alpha1/kfservice_defaults.go

```diff
@@ -7,7 +7,7 @@ import (
 
 // Default Values
 var (
-	DefaultTensorflowVersion  = "1.12"
+	DefaultTensorflowVersion  = "1.13.0"
```

Comment:
Maybe change to DefaultTensorflowServingVersion since that is what this refers to? (TF is only at 1.12.2 - https://github.com/tensorflow/tensorflow/releases)

Reply:
Make sense, although they announced to support REST in 1.12 but I can only get it working on 1.13.0 for some reason.

Reply:
Odd. I believe REST was unofficially supported starting in 1.11.